homeassistant/components/microbees/__init__.py

```diff
@@ -0,0 +1,69 @@
+"""The microBees integration."""
+
+from http import HTTPStatus
+import logging
+
+from aioaseko import dataclass
+import aiohttp
+from microBeesPy.microbees import Bee, MicroBees
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
+from homeassistant.helpers import config_entry_oauth2_flow
+
+from .const import ACCESS_TOKEN, DOMAIN, PLATFORMS
+from .coordinator import MicroBeesUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass(frozen=True, kw_only=True)
+class HomeAssistantMicroBeesData:
+    """Microbees data stored in the Home Assistant data object."""
+
+    bees: list[Bee]
```

Comment:
Also a nice question, why do we store the bees if the coordinator already has them?